website/docs/d/backup_compliance_policy.html.markdown

```diff
@@ -0,0 +1,168 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: backup_compliance_policy"
+sidebar_current: "docs-mongodbatlas-datasource-backup-compliance-policy"
+description: |-
+    Provides a Backup Compliance Policy Datasource.
+---
+
+# Data Source: mongodbatlas_backup_compliance_policy
```

Comment:
After we sort out the resource docs page issues and functionality over all this will need to be updated.  Not going into details here as it will be redundant to resource.